docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[detection-entity-dashboard]]
+= Entity dashboard
+
+The Entity Analytics dashboard provides a centralized view into emerging insider threats - including host risk, user risk, and notable anomalies from within your network. Use these analytics to triage emerging threats, investigate, and respond.
+
+
+.Requirements
+[sidebar]
+--
+The following is required to display the Entity Analytics dashboard: 
+
+* A https://www.elastic.co/pricing/[Platinum subscription] or higher.
+* To display host and user risk scores, the host risk score and user risk score features must be enabled. You can do this directly from the dashboard. For more information, refer to Enable host risk score and <<deploy-user-risk-score, Enable user risk score>>.
+* To display notable anomalies, the following machine learning jobs must be {ml-docs}/ml-ad-run-jobs.html[installed and run]: 
+** `auth_rare_source_ip_for_a_user`
+** `suspicious_login_activity`
+** `packetbeat_dns_tunneling`
+** `packetbeat_rare_server_domain`
+** `packetbeat_rare_dns_question`
+** `v3_windows_anomalous_script`
+--
+
+
+The dashboard includes the following sections:
+
+* Entity KPIs 
+* Host risk scores
+* User risk scores
+* Notable anomalies 
+
+
+[role="screenshot"]
+image::images/entity-dashboard.png[Entity dashboard]
+
+*Entity KPIs (key performance indicators)*
+
+This chart displays the total number of critical hosts, critical users, and anomalies. Select a link to go to the Host risk table, User risk table, or Anomaly Detection Jobs page. 
+
+*Host risk scores*
+
+This table displays host risk score analytics in your environment, including the total number of hosts, the five most recently recorded host risk scores, their host names, and host risk classification. Host risk scores are calculated using a weighted sum on a scale of 0 to 100. 
```

Comment:
```suggestion
This table displays host risk score data for your environment, including the total number of hosts, and the five most recently recorded host risk scores, with their associated host names and risk classifications. Host risk scores are calculated using a weighted sum on a scale of 0 to 100. 
```

Reply:
+1, and should we specify that higher score = higher risk? Or is that kinda obvious? I go either way on that

Reply:
@joepeeples I think we can assume 0-100 is low-high, but maybe I can say `0 (lowest) to 100 (highest).`